apps/src/StudioApp.js

```diff
@@ -1,5 +1,14 @@
-// Globals:
-//   Blockly
+/* jshint
+  strict: true,
+  curly: true,
+  funcscope: true,
```

Comment:
This option suppresses warnings about declaring variables inside of control structures while accessing them later from the outside.